nautobot/core/management/commands/generate_performance_test_endpoints.py

```diff
@@ -204,16 +203,20 @@ def construct_view_name_and_url_pattern(self, pattern) -> tuple[Optional[str], O
                 app_name = f"{app_name}-api"  # dcim-api
                 view_name = f"{app_name}:{pattern.name}"  # dcim-api:device-list
             else:
+                api_app_name = f"{app_name}-api"  # example_app-api
+                view_name = (
+                    f"plugins-api:{api_app_name}:{pattern.name}"  # plugins-api:example_app-api:examplemodel-list
+                )
+                app_name = app_name.replace("_", "-")  # example_app -> example-app
                 url_pattern = f"/api/plugins/{app_name}/{pattern.pattern}"  # /api/plugins/example-app/models/
-                app_name = f"{app_name}-api"  # example_app-api
-                view_name = f"plugins-api:{app_name}:{pattern.name}"  # plugins-api:example_app-api:examplemodel-list
         else:
             if not is_app:
                 url_pattern = f"/{app_name}/{pattern.pattern}"  # /dcim/devices/
                 view_name = f"{app_name}:{pattern.name}"  # dcim:device_list
             else:
-                url_pattern = f"/plugins/{app_name}/{pattern.pattern}"  # /plugins/example-app/models/
                 view_name = f"plugins:{app_name}:{pattern.name}"  # plugins:example_app:examplemodel_list
+                app_name = app_name.replace("_", "-")  # example_app -> example-app
```

Comment:
Can you test this after installing something like the floor plan or bgp-models app? Thinking of `nautobot_floor_plan` vs `floor-plan`, this may not be flexible/"intelligent" enough. Fine to address in a follow on PR if needed though.

Reply:
I will open up a follow-up issue on this.

Reply:
https://github.com/nautobot/nautobot/issues/6858